projects/ossf-scorecard/project.yaml

```diff
@@ -0,0 +1,11 @@
+homepage: "https://openssf.org/"
+main_repo: "https://github.com/ossf/scorecard"
+primary_contact: "hey@auggie.dev" # <-- IAM / commits: foo@auggie.dev
+auto_ccs:
+  - azeemshaikh38@gmail.com
+  - naveensrinivasan@protonmail.com
```

Comment:
p.s. I only added the emails I could quickly glean, but for other scorecard maintainers, feel free to PR yourselves in here w/ your preferred email address.